feature-libs/product-configurator/rulebased/occ/variant/converters/occ-configurator-variant-normalizer.spec.ts

```diff
@@ -373,16 +373,22 @@ describe('OccConfiguratorVariantNormalizer', () => {
       expect(result.immediateConflictResolution).toBe(false);
     });
 
-    it('should convert "newConfiguration" setting', () => {
+    it('should convert "newConfiguration" setting to true', () => {
       configuration.newConfiguration = true;
       const result = occConfiguratorVariantNormalizer.convert(configuration);
       expect(result.newConfiguration).toBe(true);
     });
 
-    it('should convert "newConfiguration" setting by default to true', () => {
+    it('should convert "newConfiguration" setting to false', () => {
+      configuration.newConfiguration = false;
+      const result = occConfiguratorVariantNormalizer.convert(configuration);
+      expect(result.newConfiguration).toBe(false);
+    });
+
+    it('should convert "newConfiguration" setting by default to undefined', () => {
```

Comment:
If you set: 
`newConfiguration: source.newConfiguration ?? false`
then
you don't need this test anymore

Reply:
see my previous comment